packages/openai/src/openai-chat-options.ts

```diff
@@ -118,6 +118,13 @@ export const openaiProviderOptions = z.object({
    * @default 'auto'
    */
   serviceTier: z.enum(['auto', 'flex']).optional(),
+
+  /**
+   * Whether to use strict JSON schema validation.
+   *
+   * @default true
```

Comment:
Is this accurate? The rest of the PR seems to be the opposite.

Reply:
no needs updating good catch